cmd/medco-loader/client.go

```diff
@@ -26,32 +26,32 @@ import (
 func loadV0(c *cli.Context) error {
 
 	// data set file paths
-	clinicalOntologyPath := c.String("ont_clinical")
-	genomicOntologyPath := c.String("ont_genomic")
-	clinicalFilePath := c.String("clinical")
-	genomicFilePath := c.String("genomic")
-	groupFilePath := c.String("group")
-	entryPointIdx := c.Int("entryPointIdx")
-	sensitiveFilePath := c.String("sensitive")
-	replaySize := c.Int("replay")
-	outputPath := c.String("output")
+	clinicalOntologyPath := c.String(optionOntologyClinical)
+	genomicOntologyPath := c.String(optionOntologyGenomic)
+	clinicalFilePath := c.String(optionClinicalFile)
+	genomicFilePath := c.String(optionGenomicFile)
+	groupFilePath := c.String(optionGroupFile)
+	entryPointIdx := c.Int(optionEntryPointIdx)
+	sensitiveFilePath := c.String(optionSensitiveFile)
+	replaySize := c.Int("replay") // TODO: no such option in the loader. Add there? Delete here?
```

Comment:
It's used before the loader is invoked actually, the replay is done directly in the dataset file. You can remove the todo.